go.mod

```diff
@@ -23,8 +23,9 @@ require (
 )
 
 require (
-	github.com/RHsyseng/operator-utils v1.4.11
+	github.com/RHsyseng/operator-utils v1.4.12
```

Comment:
@davidesalerno seems that operator-utils is being used already :)

Reply:
Yes we introduced it after the latest release that makes it working with our kubernetes libraries ;-)